Remove internal libgtk-x11-2.0.so checks.

upx -d animechecker
base64 animechecker > animechecker.b64
patch -p1 < animechecker.b64.patch
base64 -d animechecker.b64 > animechecker-patched
chmod a+x animechecker-patched

sha256:
4707e94a73a136b87e8dc744b667a731b5f473765f2e97cad9cf68d7b4a56c81 *animechecker-patched

--- a/animechecker.b64
+++ b/animechecker.b64
@@ -19163,7 +19163,7 @@
 Zm9jdXMAaG9yaXpvbnRhbC1zZXBhcmF0b3IAaW5jb25zaXN0ZW50AGludGVyaW9yLWZvY3VzAG1v
 ZGUAcGl4YnVmAHRleHQAeGFsaWduAHlwYWQAb3V0cHV0LXVyaQAAZ3RrLWNhbmNlbAAAZ3RrLW9r
 AAAmKxUIAAAAAGR3dC5pbnRlcm5hbC5ndGsuT1MuT1MAAAAAAAAAAAAAmAQfCDQsEQhELBEITCwR
-CGAsEQhkd3QuaW50ZXJuYWwuZ3RrLk9TAAAAAAAAAAAAAAAAAGxpYmd0ay14MTEtMi4wLnNvAAAA
+CGAsEQhkd3QuaW50ZXJuYWwuZ3RrLk9TAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
 EQAAAForFQguMAAAAAAAAAIAAAB2KxUIQ291bGQgbm90IGxvYWQgdGhlIGxpYnJhcnkge30AAAAd
 AAAAhisVCGlzb19hMwBpc29fYTQAaXNvX2E1AGlzb19iNQBuYV9sZXR0ZXIAbmFfZXhlY3V0aXZl
 AG5hX2xlZ2FsAHByaW50ZXIAb3JpZW50YXRpb24AcGFwZXItZm9ybWF0AHBhcGVyLXdpZHRoAHBh
@@ -22596,8 +22596,8 @@
 AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
 AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
 AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAZHd0Lmlu
-dGVybmFsLmMuWHV0aWwAZHd0LmludGVybmFsLmMucGFuZ29jYWlybwAAAAAAAAAAAAAAAABsaWJn
-dGsteDExLTIuMC5zbwAAABEAAADLJxgILjAAAAAAAAACAAAA5ycYCHt9OiBTeW1ib2wgJ3t9JyBu
+dGVybmFsLmMuWHV0aWwAZHd0LmludGVybmFsLmMucGFuZ29jYWlybwAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAABEAAADLJxgILjAAAAAAAAACAAAA5ycYCHt9OiBTeW1ib2wgJ3t9JyBu
 b3QgZm91bmQAAAAAAAAAGQAAAPcnGAhDb3VsZCBub3QgbG9hZCB0aGUgbGlicmFyeSB7fQAAAB0A
 AAAfKBgIZHd0LmludGVybmFsLmMuZ3RrX3VuaXhfcHJpbnRfMl8wLkd0a1ByaW50Q2FwYWJpbGl0
 aWVzAAABAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
